crates/stc_ts_type_checker/tests/conformance.pass.txt

```diff
@@ -1015,6 +1015,8 @@ es6/spread/iteratorSpreadInCall2.ts
 es6/spread/iteratorSpreadInCall3.ts
 es6/spread/iteratorSpreadInCall4.ts
 es6/spread/iteratorSpreadInCall5.ts
+es6/spread/iteratorSpreadInCall8.ts
```

Comment:
This is the cause

Reply:
oh........ :man_facepalming: no idea how I did that, thanks!